torch/nn/modules/rnn.py

```diff
@@ -206,7 +205,7 @@ def permute_hidden(self, hx: Tensor, permutation: Optional[Tensor]):
             return hx
         return apply_permutation(hx, permutation)
 
-    def forward(self, input: Tensor, hx: Optional[Tensor] = None) -> Tuple[Tensor, Tensor]:
+    def forward(self, input: Tensor, hx: Optional[Tensor] = None) -> Tuple[Union[Tensor, PackedSequence], Tensor]:
```

Comment:
`input` can also be a PackedSequence, right?

Reply:
Yes, you're right